recipes/delta-spark/LICENSE.txt

```diff
@@ -0,0 +1,388 @@
+Copyright (2021) The Delta Lake Project Authors.  All rights reserved.
```

Comment:
Since you're pulling from GitHub, this file can probably be removed from here as it will be in the GitHub tar ball.